galata/test/jupyterlab/file-search.test.ts

```diff
@@ -113,6 +113,59 @@ test.describe('File search from selection', () => {
     ).toHaveCount(2);
   });
 
+  test('search highlighting', async ({ page }) => {
+    const textToType = `True, False, def, assert # keywords
+  1, 0.1, 0x21 # numbers
+  'a', "b", "f", "c" # strings
+  bool, int, open, help # builtins
+  True, False, def, assert # keywords
+  1, 0.1, 0x21 # numbers
+  'a', "b", "f", "c" # strings
+  bool, int, open, help # builtins
```

Comment:
Thinking about this, it would reduce the chance of flaky tests if we remove this part (i.e. keep one yellow chunk and one black):

```suggestion
```